### ISSUE TITLE
Update installer for vSphere 7.0

### ISSUE THREAD

**Issue by eroji** [2vcps/cns-installer#1]
Thank you for writing this installer. It's much better than the laborious and vague instructions VMware has made available. For the most part it is deploying for me ok. I'm using Rancher and the taints are slightly different. However, I am testing it with vSphere 7.0 and looks like your CSI controller deploy and daemonsets are both not the latest version. The controller is complaining about the API version 7.0 is too new. Is it possible to update this for vSphere 7.0?

**Comment by 2vcps** [comment]
Let me see what i can do. Just now getting my 7.0 env up.

**Comment by eroji** [comment]
I forked your repo and managed to get it working. It was mostly a matter of pulling the latest set of tags for the components from VMware's registries. I'm using Rancher so I had to make some modifications for the taint and tolerance. It is otherwise working.

**Comment by 2vcps** [comment]
Yes, Rancher's taints makes it a challenge Glad you got it working. If you create a PR with the fixes I will test to merge it.
